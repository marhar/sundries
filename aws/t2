#!/bin/sh
rm -f t2.out
(curl -s 'http://localhost:8888/math?op=add&x=4&y=5'
curl -s 'http://localhost:8888/math?op=sub&x=4&y=5'
curl -s 'http://localhost:8888/math?op=mul&x=4&y=5'
curl -s 'http://localhost:8888/math?op=div&x=4&y=5') >t2.out
diff t2.out t2.expected
if test $? -eq 0; then
    rm t2.out
fi
