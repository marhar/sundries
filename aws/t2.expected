9
-1
20
0
